app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    """Base class for vaccine errors"""
+
+
+class NotVaccinatedError(VaccineError):
+    """No vaccine certificate were provided"""
```

Comment:
The docstring for `NotVaccinatedError` contains a grammatical error. It should be 'No vaccine certificate was provided' instead of 'No vaccine certificate were provided'.